src/Tribe/PUE/Checker.php

```diff
@@ -564,16 +564,11 @@ public function display_license_error_message() {
 		}
 
 		public function get_license_expired_message() {
-			$helper_text = '<span class="screen-reader-text">' . __( ' (opens in a new window)', 'tribe-common' ) . '</span>';
-			return sprintf(
-				__(
-					'This license key is expired! You need to %1$srenew your license %2$s %3$s to get access to the latest versions including bug fixes, security updates, and new features.',
-					'tribe-common'
-				),
-				'<a href="http://m.tri.be/195y" target="_blank">',
-				$helper_text,
-				'</a>'
-			);
+			$expired_message = '<a href="http://m.tri.be/195y" target="_blank" class="button button-primary">' . __( 'Renew Your License Now', 'tribe-common' ) . '</a>';
+
+			$expired_message .= '<span class="screen-reader-text">' . __( ' (opens in a new window)', 'tribe-common' ) . '</span>';
```

Comment:
The intent is to have the `$expired_message` span _inside_ the anchor tag, I believe. @trishasalas can correct me if I'm wrong.